src/extras/helpers/PolarGridHelper.js

```diff
@@ -0,0 +1,94 @@
+import { LineSegments } from '../../objects/LineSegments';
+import { VertexColors } from '../../constants';
+import { LineBasicMaterial } from '../../materials/LineBasicMaterial';
+import { Float32Attribute } from '../../core/BufferAttribute';
+import { BufferGeometry } from '../../core/BufferGeometry';
+import { Color } from '../../math/Color';
+import { _Math as Math } from '../../math/Math';
```

Comment:
We use Math.sin and Math.cos, do you mean we don't need to do `_Math as Math` and just import `Math`?